earth_enterprise/rpms/build.gradle

```diff
@@ -262,13 +263,14 @@ task openGeeCommonRpm(type: GeeRpm, dependsOn: openGeeSharedFiles) {
     autoFindProvides = true
     autoFindRequires = true
 
-    requires('opengee-postgis', '1.5.8', GREATER | EQUAL)
     requires('chkconfig')
     requires('shadow-utils')
     requires('perl')
     requires('sudo')
-    conflicts('opengee-postgis', '2.0', GREATER | EQUAL)
-
+    requires('xerces-c')
+    requires('ogdi')
+    requires('geos')
```

Comment:
Unnecessary:

```bash
rpm -qpR opengee-common-5.2.0-1.el7.x86_64.rpm | grep geos

libgeos_c.so.1()(64bit)
```

Reply:
On different versions of RHEL6 with satellite servers these dependencies have not always been auto-detected.  More testing should be done before removing these explicit dependencies - I recommend that we leave them in for the time being as a safeguard.